widget/templates/css/_style.scss

```diff
@@ -0,0 +1,10 @@
+.my-widget {
```

Comment:
I think this needs to be `.<%= baseClass %> {` because I created a widget called SassTest and the output .scss file has `.my-widget` instead of `.sass-test`.